pkg/networkpolicy/controller.go

```diff
@@ -183,9 +204,16 @@ func newController(client clientset.Interface,
 		return nil, err
 	}
 
+	pods := 0
 	// process only local Pods that are affected by network policices
-	_, _ = podInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc: func(obj interface{}) {
+	_, _ = podInformer.Informer().AddEventHandler(cache.ResourceEventHandlerDetailedFuncs{
+		AddFunc: func(obj interface{}, isInitialList bool) {
+			if isInitialList {
+				pods++
+				if pods%200 == 0 {
+					runtime.GC()
```

Comment:
this scares me

Reply:
it doesn't scare me but it seems kind of ugly and maybe would fit better in the the workqueue-processing code (eg `processNextItem`) rather than the informer itself?

Reply:
Maybe GOMEMLIMIT  would be a more conventional approach?